pages/participantinfo.tsx

```diff
@@ -404,48 +996,106 @@ export default function ParticipantInfo({
-        } else if (pageNum === totalPages - 1) {
-            return (
-                <FormButton
-                    onClick={() => setPageNum((prevPage) => prevPage + 1)}
-                >
-                    Finish
-                </FormButton>
-            );
-        } else if (pageNum === proofOfIncomePage) {
-            return (
-                <Box>
-                    <HStack spacing="24px">
-                        <FormButton>Upload Proof of Income</FormButton>
-                        <Button
-                            variant="ghost"
-                            as="u"
-                            onClick={() =>
-                                setPageNum((prevPage) => prevPage + 1)
-                            }
-                            borderRadius={100}
-                        >
-                            Skip for Now
-                        </Button>
-                    </HStack>
-                </Box>
-            );
-        }
-        return (
-            <FormButton
-                onClick={() => {
-                    setPageNum((prevPage) => prevPage + 1);
-                    window.scrollTo({ top: 0 });
-                }}
-            >
-                Next
-            </FormButton>
-        );
+    async function updateUser() {
+        const parentData: ParentInput = {
+            phoneNumber: parentPhoneNumber,
+            isLowIncome: undefined, // TODO
+            preferredLanguage: locale.en,
+            proofOfIncomeLink: undefined, // TODO
+            heardFrom: heardFromOptions.sort(),
+
+            childFirstName: participantFirstName,
+            childLastName: participantLastName,
+            childDateOfBirth: new Date(dateOfBirth),
+            addressLine1: address1,
+            addressLine2: address2,
+            postalCode: postalCode,
+            cityName: city,
+            province: participantProvince,
+            school: school,
+            grade: parseInt(grade, 10),
+
+            difficulties: participantDifficulties.sort(),
+            otherDifficulties: hasOtherDifficulties ? otherDifficulties : null,
+            specialEducation: specialEd,
+            therapy: participantTherapy.sort(),
+            otherTherapy: hasOtherTherapy ? otherTherapy : null,
+
+            guardianExpectations,
+            emergencyContactFirstName: emergFirstName,
+            emergencyContactLastName: emergLastName,
+            emergencyContactPhoneNumber: emergNumber,
+            emergencyContactRelationToStudent: emergRelationship,
+            medication: isOnMedication ? medication : null,
+            allergies: hasAllergies ? allergies : null,
+        };
+        const userData = {
+            firstName: parentFirstName,
+            lastName: parentLastName,
+            role: roles.PARENT,
+            roleData: parentData,
+        };
+        const request = {
+            method: "PUT",
+            headers: { "Content-Type": "application/json" },
```

Comment:
Do you need to specify this? If so, maybe we can add a util class for making requests similar to how we have `ResponseUtil` for the backend!

Reply:
I think it is - when I exclude it I get a `500` error. Should I make that in this PR?